src/screens/Settings.js

```diff
@@ -51,7 +52,11 @@ class Settings extends React.Component {
    * When user clicks Add Passphrase button we redirect to Passphrase screen
    */
   addPassphrase = () => {
-    this.props.history.push('/wallet/passphrase/');
+    if (hathorLib.wallet.isHardwareWallet()) {
+      $('#notSupported').modal('show');
```

Comment:
I think here we should say that the passphrase must be set in the hardware wallet.

Reply:
Ok, now it's possible to pass a custom text to the modal. In this case, I'm telling users to set the passphrase on Ledger.

Reply:
I think it should show the "not supported" message. It is supported but it must be done on Ledger.

Reply:
I don't understand what you mean. We currently show this when the user clicks 'Set a passphrase':
![image](https://user-images.githubusercontent.com/3236756/74266845-440b7c00-4d05-11ea-9ddb-d13649c7e851.png)

Reply:
I think the title can be misleading, and we should change it to "Action supported on your hardware wallet".

Reply:
It's not supported on the wallet itself, only on the hardware wallet, so the title doesn't seem misleading to me.

What you suggested seems like a weird title for me, to be honest. If we want to change, I'd go for something like "Complete action on your hardware wallet"

Reply:
I like your suggestion "Complete action on your hardware wallet".

Reply:
Done